nvflare/private/fed/server/server_engine.py

```diff
@@ -317,8 +317,6 @@ def abort_app_on_clients(self, clients: List[str]) -> str:
         return ""
 
     def abort_app_on_server(self, job_id: str, turn_to_cold: bool = False) -> str:
-        if job_id not in self.run_processes.keys():
```

Comment:
can you add a log message here to help understanding the sequence of event. This is a very unusual case.

Reply:
Added log message to show the abort SJ job which not supposed to be running.

Reply:
So we don't need this check anymore here?

Reply:
We use the same abort_app_on_server() to abort the job which is not in the parent's running job list. In this case, within abort_app_on_server() we don't need to check this any more. The checking logic is at where calls this method.